packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -267,6 +279,22 @@ describe('Sayt Component', () => {
     });
   });
 
+  describe('getDebounce()', () => {
+    it('should return a debounced callback method', () => {
+      const delay = sayt.debounce;
+      const debounceSettings = false;
+      const termsCallback = stub(sayt, 'requestSaytAutocompleteTerms');
+      const debounceStub = stub(Debounce, 'debounce');
+      const expectedDebouncedFunction = () => 123;
+
+      debounceStub.returns(expectedDebouncedFunction);
```

Comment:
This should be part of the arrange (first) section of the test. The stub is created in the test so there's no need to change it later.